pod/pod-api-public-deprecated.yaml

```diff
@@ -6635,7 +6635,9 @@ definitions:
     properties:
       id:
         type: string
-        format: long
+        enum: [INDIVIDUAL, SUPER_ADMINISTRATOR, L1_SUPPORT, L2_SUPPORT, COMPLIANCE_OFFICER,
```

Comment:
adding enum let up keep control on the id value, but it needs to be maintained if any change is happened in Roles Object ids. Is it a good idea to keep it/remove it ?